packages/cmrjs/index.js

```diff
@@ -77,15 +77,15 @@ async function ingestConcept(type, xml, identifierPath, provider, token) {
     });
   });
 
-  //log.debug('XML object parsed', logDetails);
+  log.debug('XML object parsed', logDetails);
```

Comment:
If this (and the next few changes) were commented out before, should they be uncommented here?